tests/test_regression.py

```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env python
 
 import os
+import os.path
```

Comment:
I don't think you need this import (you explicitly call `os.path.splitext` later)